common/util_test.go

```diff
@@ -90,6 +90,6 @@ func TestRatToFixed(t *testing.T) {
 
 	// more than 3 decimals returns an error and 0
```

Comment:
We should update this comment too since we're no longer returning an error for this case

Reply:
adjusted in https://github.com/livepeer/go-livepeer/pull/939/commits/c4631f7cd9d9706f31bdfb76981c380796b98ec0